src/tablo/Column/Column.js

```diff
@@ -16,6 +16,7 @@ const argsTypes = struct({
   data: maybe(t.Array),
   name: t.String,
   fixed: maybe(t.Boolean),
+  flexGrow: maybe(t.Number),
```

Comment:
we should consider making this a `union([t.Boolean, t.Number])` to make it work like `FlexView` (`flexGrow={true} ---> flexGrow: 1`)